packages/client/hmi-client/src/services/workflow.ts

```diff
@@ -250,10 +250,19 @@ export class WorkflowWrapper {
 			width: nodeSize.width,
 			height: nodeSize.height
 		};
-		if (op.initState && _.isEmpty(node.state)) {
+		if (op.initState) {
 			node.state = op.initState();
+			Object.assign(node.state, options.state);
 		}
 		this.wf.nodes.push(node);
+
+		if (options.outputValue) {
+			node.status = OperatorStatus.SUCCESS;
+			const outputPort = node.outputs[0];
+			outputPort.value = isArray(options.outputValue) ? options.outputValue : [options.outputValue];
+			node.active = outputPort.id;
+			this.selectOutput(node, outputPort.id);
+		}
```

Comment:
It might be better to move this `outputValue` out of addNode(...), as this mechanism would only work for a couple of operators, the remaining operators need to more setup like output-state.

eg in situational-awareness createWorkflow, this sequence should create a workflow with the correct data
```
const model  = wf.addNode(...)
const modelConfig = wf.addNode(...)
const simulate = wf.addNode(...)

// .. add edges

Object.assign(model.outputs[0], ...) // state, value, status, operatorStatus
wf.selectOutput(model, model.outputs[0].id)

Object.assign(modelConfig.outputs[0], ...) // state, value, status, operatorStatus
wf.selectOutput(modelConfig, modelConfig.outputs[0].id)

Object.assign(simulate.outputs[0], ...)
wf.selectOutput(simulate, simulate.outputs[0].id) // state, value, status, operatorStatus

```